sources/packages/backend/apps/api/src/services/index.ts

```diff
@@ -32,6 +32,7 @@ export * from "./student-assessment/student-assessment.constants";
 export * from "./student-appeal/student-appeal.service";
 export * from "./student-appeal-request/student-appeal-request.service";
 export * from "./student-scholastic-standings/student-scholastic-standings.service";
+export * from "./student-scholastic-standings/student-scholastic-standings.models";
```

Comment:
Can we move the import before the service at line 34. Alternatively removing this line will also work as I do not see an import of any model in the PR.